package.json

```diff
@@ -33,6 +33,7 @@
     "grunt-docco2": "0.2.0",
     "grunt-shell-spawn": "0.3.1",
     "jshint": "2.6.3",
+    "mocha": ">= 1.20.0",
```

Comment:
@jh-bate @jebeck I'm fine with choosing a specific version. We can choose the latest one it installs right now (2.2.5). The reason I added it this way was because this specific dependency was being pulled in implicitly (rather than explicitly) from the inclusion of grunt-mocha-test (and implicit peer dependencies will be deprecated in the future).

Is this something you want me to do?